src/components/ChooseCheckGroup/CheckGroupCard.tsx

```diff
@@ -99,10 +102,57 @@ function getTooltip(
 }
 
 const getStyles = (theme: GrafanaTheme2) => ({
+  newBadge: css({
+    position: 'absolute',
+    right: 0,
+    marginRight: theme.spacing(3),
+    marginTop: theme.spacing(2),
+    height: '26px',
+  }),
+
+  cardHeader: css({
+    maxHeight: '20px',
+  }),
+
+  cardDescription: css({
+    maxHeight: '70px',
+    overflow: 'hidden',
+  }),
+
+  cardButton: css({
+    marginTop: 'auto',
+  }),
+
+  cardFooter: css({
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    gap: '4px',
+    maxHeight: '20px',
+  }),
+
+  checkCard: css({
+    minWidth: '0',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'wrap',
+
+    '> div:first-of-type': {
+      height: '100%',
+    },
+  }),
   desc: css({
     color: theme.colors.text.secondary,
   }),
+  groupName: css({
+    color: theme.colors.text.primary,
+  }),
   protocols: css({
-    marginTop: theme.spacing(1),
+    borderTop: `1px solid ${theme.colors.border.weak}`,
+    color: theme.colors.text.primary,
+    display: 'flex',
+    height: '35px',
```

Comment:
Same with is this is needed? See the image below.

![The API Endpoint card. The protocols are being cut off when they break to two lines.](https://github.com/user-attachments/assets/d91e965d-9f5c-4394-a180-98b6873702b0)